scripts/arango/test-data/testData.js

```diff
@@ -0,0 +1,75 @@
+const testData = [
```

Comment:
can we not import json? It feels awkward to have to define and export a const, but it's not th end of the world. What I'd really like to do is write this test data using TypeScript, so it's easy to get the types right. We should look at if there's a way to require developers to create test data for every new model. Very happy you got this to load from a separate file. Awesome work!